src/cloud_provider/digitalocean/kubernetes/mod.rs

```diff
@@ -1100,6 +1101,25 @@ impl<'a> DOKS<'a> {
             }
         }
 
+        info!("Delete Qovery managed object storage buckets");
+        if let Err(e) = self.spaces.empty_bucket(self.kubeconfig_bucket_name().as_str()) {
+            return Err(EngineError::new(
+                EngineErrorCause::Internal,
+                self.engine_error_scope(),
+                self.context().execution_id(),
+                e.message,
+            ));
+        }
+
+        if let Err(e) = self.spaces.empty_bucket(self.logs_bucket_name().as_str()) {
```

Comment:
Should we trash logs ? Don't we want to keep those ?

Reply:
It's on cluster deletion. In Scaleway, it's deleted, that's why I reproduce it there but without deleting buckets (DO issue, you know). But yeah keeping logs could be an idea.

Reply:
Ok ok ! Let's be consistent then ;)

Reply:
Done !